### ISSUE TITLE
gauge --format --verbose does not give any information

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#477]
### Expected behavior

Should log the information
### Actual behavior

It does not give any info
### Steps to reproduce
1. Create a project

```
gauge --init ruby
```
1. gauge --format --verbose
### Gauge version

```
Gauge version: 0.5.1.nightly-2016-07-10

Plugins
-------
html-report (2.1.1.nightly-2016-07-08)
java (0.4.2.nightly-2016-07-07)
ruby (0.2.1)
```

**Comment by apoorvam** [comment]
The flag `--verbose` is to enable step level reporting on console during execution. If detailed logs are needed, `--log-level=DEBUG` flag can be set. 

@sguptatw What is the kind of information that is expected on spec format?

**Comment by sguptatw** [comment]
Consider formatting a group of specs
If a x number of specs get changed due to the command, I would like to know the names of such files when I use the `verbose` mode only.

**Comment by apoorvam** [comment]
With `gauge --log-level=debug --format specs` you should now get detail info on what specs are formatted and errors, if any.

Fix should be available in nightly>=7.12.2016
